cycode/cyclient/models.py

```diff
@@ -171,6 +171,23 @@ def __init__(
         self.err = err
 
 
+class ScanReportUrlResponse(Schema):
+    def __init__(
+        self,
+        report_url: str,
+    ) -> None:
+        super().__init__()
```

Comment:
pls use `@dataclass` instead. this is legacy approach. see the examples at the end of `models.py` file